inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -20,6 +20,8 @@
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 import org.apache.inlong.sort.base.sink.SchemaUpdateExceptionPolicy;
+import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY;
```

Comment:
Hi, the `inlong-sort` module should not dependency the `inlong-manager` module directly, can you move the `org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `inlong-common` module?

Reply:
Hi @healchow 
Have taken  care of this in next commit. Please let me know if the changes look good?

Reply:
LGTM